Readme.md

```diff
@@ -41,6 +41,8 @@ You can then easily install all of the dependencies with one of the following co
 
     $ brew bundle
 
+In case dependency already installed and there is update available it will be upgraded.
```

Comment:
`If a dependency is already installed and there is an update available it will be upgraded.`